processor/src/main/java/com/linecorp/decaton/processor/runtime/internal/QuotaApplierImpl.java

```diff
@@ -71,7 +70,7 @@ public boolean apply(ConsumerRecord<byte[], byte[]> record,
             try {
                 topic = callback.apply(record, quotaUsage.metrics());
             } catch (Exception e) {
-                log.error("Exception thrown from the quota callback for key: {}", Arrays.toString(record.key()), e);
+                log.error("Exception thrown from the quota callback", e);
```

Comment:
don't we want to log the record offset maybe instead?

Reply:
It depends on the QuotaCallback implementation though, IMO logging the exception is enough to tell what happened.
Users can add logging inside QuotaCallback implementation if necessary.

Reply:
in the first place the user can optionally log more information through enabling format with MDC values?